js/damage_dpp.js

```diff
@@ -1,4 +1,5 @@
 ﻿function CALCULATE_ALL_MOVES_DPP(p1, p2, field) {
+	var SP;
```

Comment:
This is _not_ how you localize a global variable...........

Reply:
Is this just a global string constant? Just actually list it as global in eslintrc

Reply:
Yeah I noticed that yesterday when I rebased. The reason it isn’t already was because you didn’t want too many globals, but yeah this declaration is just wrong. The other stats will also have to be fixed

Reply:
This doesn't _stop_ it from being a global.

Honestly, I don't mind having the six stat constants as globals. It's probably easier than whatever other plan you have for them.

Reply:
Yeah that’s what’s i noticed lol i definetly got confused somewhere down the road as originally they were listed as globals but you requested them to be changed at the time but anyways will change

Reply:
I said you shouldn't make _everything_ a global. These seem to make more sense as globals.